index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Apple old website clone.</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+</head>
+<body>
+  <header>
+    <nav>
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span class="iconify apple-logo" data-icon="ant-design:apple-filled" data-inline="false"></span>
+        </a>
+      </span>
+
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span>Store</span>
+        </a>
+      </span>
+
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span>iPod</span>
+        </a>
+      </span>
+
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span>iPhone</span>
+        </a>
+      </span>
+
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span>iPad</span>
+        </a>
+      </span>
+
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span>iTunes</span>
+        </a>
+      </span>
+
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span>Support</span>
+        </a>
+      </span>
+
+      <form action="self" class="search-form">
+        <label for="search" id="search-label">
+          <span class="iconify" data-icon="bx:bx-search" data-inline="false">
+          </span>
+        </label>
+        <input type="search" name="search" id="search">
+      </form>
+
+    </nav>
+    <div class="banner-text">
+      <figure class="text-img-top">
+        <a href="#">
+          <img src="images/what.png" alt="" >
```

Comment:
Images are missing alternative text. Please fill it with the appropriate information.